chcemvediet/static/main/css/typo.sass

```diff
@@ -88,6 +88,17 @@ a, .btn-link
   &:focus, &:hover
     text-decoration: none
 
+//
+// Footnote style
+//
+
+.chv-footnote
```

Comment:
Toto nepatri do `typo.sass`. Nie je to vseobecna typografia, ale konkretny komponent. Vhodny subor je asi `custom.sass`, kde mame rozne custom komponenty. Ked tam toho bude privela, mozeme to casom rozdelit do viacerych suborov.

Reply:
:ok: